김현경/모의고사.py

```diff
@@ -0,0 +1,23 @@
+def solution(answers):
+    answer = []
+    a = [1, 2, 3, 4, 5] * 2000
+    b = [2, 1, 2, 3, 2, 4, 2, 5] * 1250
+    c = [3, 3, 1, 1, 2, 2, 4, 4, 5, 5] * 1000
```

Comment:
```suggestion
def generate_pattern(pattern, length=10000):
    return (pattern * (length // len(pattern)))[:length]

def solution(answers):
    answer = []
    
    a = generate_pattern([1, 2, 3, 4, 5])
    b = generate_pattern([2, 1, 2, 3, 2, 4, 2, 5])
    c = generate_pattern([3, 3, 1, 1, 2, 2, 4, 4, 5, 5])
```

아까 구두로 얘기했던 10000개 맞추는 부분 이렇게 해도 괜찮지 않을까 싶어요! 슬라이싱 부분은 어차피 초과해도 상관없으니까 없어도 될 것 같긴한데, 그래도 10000개 딱 맞춰주는 코드로 한 번 해봤습니다! (사실 제가 짠 건 아니고.. gpt가 도와줬어용 ㅎㅎ)